src/migratus/core.clj

```diff
@@ -146,17 +151,28 @@
   [config & [name]]
   (mig/destroy config name))
 
+(defn select-migrations [config selection-fn]
+  (let [migrations (->> (doto (proto/make-store config)
+                               (proto/connect))
+                             (selection-fn config)
+                             (mapv (juxt proto/id proto/name)))]
+    migrations))
+
+(defn completed-list
+  "List completed migrations"
+  [config]
+  (let [migrations (select-migrations config completed-migrations)]
+    (log/debug (apply str "You have " (count migrations) " completed migrations:\n"
+                      (str/join "\n" migrations)))
+    (mapv second migrations)))
+
 (defn pending-list
-  "List pending migrations"
+  "List completed migrations"
```

Comment:
shouldn't this still be pending?

Reply:
Yes, it's a copy and paste error, I missed it.